Calculate/Calculator.cs

```diff
@@ -0,0 +1,57 @@
+﻿namespace Calculate;
+
+public class Calculator
+{
+    public IReadOnlyDictionary<char, Func<int, int, double>> MathematicalOperations { get; }
+    public Calculator()
+    {
+        MathematicalOperations = new Dictionary<char, Func<int, int, double>>()
+        {
+            {'/', Divide},
+            {'*', Multiply},
+            {'-', Subtract},
+            {'+', Add}
+        };
+    }
+
+    public static double Add(int paramOne, int paramTwo)
+    {
+        return paramOne + paramTwo;
+    }
```

Comment:
```suggestion
    public static double Add(int paramOne, int paramTwo) => paramOne + paramTwo;
```
You could do this in a single line, this is acceptable for simple methods that are just doing one thing.